apps/theming-designer/src/components/SemanticSlots.tsx

```diff
@@ -44,6 +44,7 @@ export const SemanticSlots: React.StatelessComponent<ISemanticSlotsProps> = (pro
     'bodySubtext', // neutralSecondary
```

Comment:
Looks like some other slots are missing that have been added in theme.ts, perhaps in previous PRs and we missed adding them in this mapping too:
`bodyBackgroundHovered` `bodyBackgroundChecked`
Might help to go through the theme.ts mapping & this one side by side to make sure they're matching fully.